lib/components/ComboList.js

```diff
@@ -130,11 +140,13 @@ function onItemsChange(msg, data) {
 }
 
 function MLComboList_get() {
-    var value = this.model.get();
+    const value = this.model.get();
     return (value && typeof value === 'object') ? _.clone(value) : value;
 }
 
 function MLComboList_set(value) {
+    const hasTooltips = value && Array.isArray(value) && !!value.find(v => v.item);
```

Comment:
is check value redundant as Array.isArray can catch undefined, if that is the case.

Reply:
fair happy to change